### PR TITLE
React-Redux Group Project.

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
- Fetched data from the SpaceX API.
- Created Routes to navigate between components.
- Created different components to display data.
- Wrote Actions and Reducers for booking, canceling rockets and dragons, and joining missions.
- Displayed all joined rockets, dragons, and missions on my profile page.
- Wrote unit and snapshot testing for all the components.
- Added styling with CSS.

### CODE REVIEW

**Comment by Somdotta07** [comment]
We didn't receive any comment from the previous reviewer. It was marked as required changes on our dashboard.

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+## Built With
+- React-Redux
+- CSS3
+
+### Technologies used
+- Github
+- Gitflow
+- Visual Studio Code
+## Live Demo
+
```
Comment (arslanbisharat) [inline README.md:21]: - [ ] I cannot find a link to the video recording of your project. Make sure you do have recorded one and linked that to Readme or Pull Request Message.

File: src/Components/tests/App.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable prefer-destructuring */
```
Comment (arslanbisharat) [inline src/Components/tests/App.test.js]: - [x] Make sure not to disable linter rules. These are here to help us write clean code.

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+table,
+td,
+th {
+  border: 1px ridge lightgray;
+  border-collapse: collapse;
+  padding: 5px 5px;
+}
+
+.mission-description {
+  /* font-family: Verdana, Geneva, Tahoma, sans-serif; */
```
Comment (arslanbisharat) [inline src/App.css]: - [x] Let's not use commented code in cdebase. this makes code unclean.

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
...
\ No newline at end of file
+# Space-Travelers-Hub
+- In this task, we will be working with the real live data from the SpaceX API. Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+# Screenshots
+
+![image](https://user-images.githubusercontent.com/87448628/146329774-2339b30f-ff2a-451f-b9a1-8c4ed53a01a5.png)
+
+![image](https://user-images.githubusercontent.com/87448628/146329869-7ff101f3-9864-4ef2-a280-248cfce284b3.png)
+
+![image](https://user-images.githubusercontent.com/87448628/146329918-5a122b3f-f6d5-4695-8791-26cf7fbc09f2.png)
```
Comment (arslanbisharat) [inline README.md]: - [x] [Optional] I would have added description of each image

File: README.md [hunk 5]
```diff
@@ -1 +1,68 @@
...
+- Gitflow
+- Visual Studio Code
+## Live Demo
+
+[Link](https://cranky-cray-0feacf.netlify.app/)
+
+## Getting Started
+
+ To clone this project :
+* Run command: $git clone [repo](https://github.com/Danieldotcomcoder/space-travelers-hub)
```
Comment (arslanbisharat) [inline README.md]: - [x] [Optional] I would recommend you to make sure that there is command instead of link.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] In the console, we can see a few errors. It would be great if you can resolve these errors. 
Capture


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by enniopo** [comment]
Hi @Somdotta07, sorry for the inconvenience caused here. 
Please, work on the changes requested at your earlier convenience, and once done please let your new CR know so he can review your work. 
Thanks and sorry again. 
Ennio

File: README.md (hunk 1 above)
Comment (Danieldotcomcoder) [reply README.md:21]: Thank you, but there is no video recording requirement for this project.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**